includes/User.php

```diff
@@ -2648,22 +2648,7 @@ public function getGlobalAttribute( $attribute, $default = null ) {
 	 * @return bool
 	 */
 	private function shouldGetAttributeFromService( $attributeName ) {
-		global $wgEnableReadsFromAttributeService;
-
-		if ( empty( $wgEnableReadsFromAttributeService ) ) {
```

Comment:
`$wgEnableReadsFromAttributeService` is universally set to true, so it's is no longer helpful. I removed this check here, and removed that variable from CommonSettings.php.